<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kabinet</class>
 <widget class="QMainWindow" name="kabinet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Личный кабинет</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitkab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #5555F5</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QLabel" name="labelkab">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>461</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите нужное вам окно</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stats_window"/>
    <widget class="QWidget" name="play_window">
     <widget class="QPushButton" name="play_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Сыграть</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="kabplay">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Окно партии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kabstats">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Окно статистики</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
